### ISSUE TITLE
Exact solutions to the nonlinear dynamics of learning in deep linear neural networks

### ISSUE THREAD

**Issue by TanUkkii007** [TanUkkii007/papers-i-read#104]
link: https://arxiv.org/abs/1312.6120

referenced from:
- Batch normalization: Accelerating deep network training by reducing internal covariate shift #74
